core/scss/components/site-search/_site-search.scss

```diff
@@ -0,0 +1,73 @@
+@charset "UTF-8";
+
+//
+// Site Search
+//
+// A global site search component
+//
+// Markup: ../templates/components/site-search/site-search.twig
+//
+// Style guide: Components.Site Search
+//
+.su-site-search__input {
+  @include font-smoothing();
+  @include padding(.3em 1em);
+
+  display: inline-block;
+  height: 40px;
+  max-width: 100%;
+  border-radius: 20px;
+  font-size: .8em;
+
+  &::placeholder {
+    color: $color-black;
+    opacity: 1; // For Firefox
+  }
+
+  @media #{$breakpoint-lg} {
+    width: 320px;
+  }
+}
+
+.su-site-search__sr-label {
+  @include sr-only;
+}
+
+.su-site-search__submit {
+  @include padding(0);
+  @include margin(0 0 0 null);
+
+  background-color: transparent;
+  opacity: .6;
+  position: absolute;
+  top: 11px;
+  right: 12px;
+  width: auto;
+
+  @media #{$breakpoint-lg} {
+    position: relative;
+    top: 7px;
+    right: 40px;
+  }
+
+  img {
+    display: inline-block;
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    background-color: transparent;
+    opacity: 1;
+  }
+
+  &:active,
+  &:focus {
+    box-shadow: none;
+  }
+
+  &:focus {
+    outline: #aaa solid 3px;
+    outline: -webkit-focus-ring-color auto 5px;
```

Comment:
Why are you changing the focus behavior just on webkit browsers? Should we specify this universally and let autoprefixer manage the vender prefix?

Reply:
It's just that webkit has their own default focus outline setting (the light fuzzy light blue) so I want it to be consistent with other elements on focus when they're using a webkit browser. There's no similar setting for other browsers. I don't believe autoprefixer deals with that particular item.